sdk/storage/azblob/blockblob/client_test.go

```diff
@@ -4680,6 +5075,72 @@ func (s *BlockBlobUnrecordedTestsSuite) TestUploadFromReader() {
 	}
 }
 
+func (s *BlockBlobUnrecordedTestsSuite) TestLargeBlockBufferedUploadInParallelWithGeneratedCRC64() {
+	_require := require.New(s.T())
+	testName := s.T().Name()
+	svcClient, err := testcommon.GetServiceClient(s.T(), testcommon.TestAccountDefault, nil)
+	_require.NoError(err)
+
+	containerName := testcommon.GenerateContainerName(testName)
+	containerClient := testcommon.CreateNewContainer(context.Background(), _require, containerName, svcClient)
+	defer testcommon.DeleteContainer(context.Background(), _require, containerClient)
+
+	bbClient := testcommon.GetBlockBlobClient(testcommon.GenerateBlobName(testName), containerClient)
+
+	var largeBlockSize, numberOfBlocks int64 = 2500 * 1024 * 1024, 2
```

Comment:
This allocation is huge and is likely why the test is failing on linux (the kernel likely just kills the process).

Reply:
This helped, thanks!